### PR TITLE
2.2.1-beta1 Release

### PR DESCRIPTION
**Pull request by outofambit**
Planning to use this branch/PR to ship a beta, prod, and another beta before merging it to `development` on Wed, Oct 9th. Please **don't update this from `development`** until we get to that point in the checklist below.

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,11 @@
 {
   "releases": {
+    "2.2.1-beta1": [
+      "[Fixed] Improperly formatted url pre-filled in \"Clone Repository\" dialog - #8394",
+      "[Fixed] VSCodium integration on macOS -  #8424. Thanks @akim8!",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Fixed] VSCodium integration on macOS - #8424. Thanks @akim8!",
```

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,11 @@
 {
   "releases": {
+    "2.2.1-beta1": [
+      "[Fixed] Improperly formatted url pre-filled in \"Clone Repository\" dialog - #8394",
+      "[Fixed] VSCodium integration on macOS - #8424. Thanks @akim8!",
+      "[Fixed] Partially hidden icon in \"Open Pull Request\" button - #8410",
+      "[Improved] Ensure crash reports are sent when app closes unexpectedly - #8405"
```
Comment (nerdneha) [inline changelog.json]: would we consider this an externally facing change?

Comment (outofambit) [reply changelog.json]: ooo that's a good point. i'll remove it

Comment (outofambit) [reply changelog.json]:
```suggestion
```

**Comment by outofambit** [comment]
this was released and its contents are part of #8437, which will be merged into `development` shortly.

**Pull request closed without merging**